README.md

```diff
@@ -93,7 +93,7 @@ your Assistant service and import your workspace. The service will be named
 
 Create the service by following this link and hitting `Create`:
 
-* [**Watson Assistant**](https://cloud.ibm.com/catalog/services/conversation)
+* [**Watson Assistant**](https://cloud.ibm.com/catalog/services/assitant)
```

Comment:
I didn't think this link changed yet, but you are correct once the typo is fixed.

Reply:
fixed